lib/classes/PluginManager.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+
+const Utils = require('./Utils');
+const path = require('path');
+const has = require('lodash').has;
+const forEach = require('lodash').forEach;
+
+class PluginManager {
+  constructor(serverless) {
+    this._serverless = serverless;
+    this._pluginInstances = [];
+    this._commandsList = [];
+    this._commands = {};
+  }
```

Comment:
I'm also more the verbose guy (see the proposed `pluginInstances` array :-D) because this way everyone knows what it's all about. I've changed `serverless` to `S` in the last commit. Should I go back to `serverless`?